environs/bootstrap/config.go

```diff
@@ -158,9 +169,12 @@ func IsBootstrapAttribute(attr string) bool {
 
 // Config contains bootstrap-specific configuration.
 type Config struct {
-	AdminSecret             string
-	CACert                  string
-	CAPrivateKey            string
+	AdminSecret  string
+	CACert       string
+	CAPrivateKey string
+	// TODO(ale8k): I've used a string, but would prefer signer type, is it ok
+	// to use other types here?
+	SSHServerHostKey        string
```

Comment:
You can use a type, add a way to get the raw bytes and the string from it.

Reply:
I did it, then wanted to change CA (so they're kind of consistent in using correct go types) and reverted back so they both stay consistently strings. We've got decent validation so I'll leave as is.

Reply:
Removing todo.